### ISSUE TITLE
Radio Button lacks focus indicator 

### ISSUE THREAD

**Issue by britt6612** [grommet/grommet#6328]
### Expected Behavior
If you `tab` to a RadioButton I would expect to have the focus on the RadioButton it self

### Actual Behavior
Receives no focus if it is not clicked 

### Steps to Reproduce

https://storybook.grommet.io/?path=/story/input-radiobutton-simple--simple
Tab here and see RadioButton never gets focused 

If you tab in this example the focus goes on the active RadioButton
https://v2.grommet.io/radiobuttongroup

### Your Environment

- Grommet version: latest
- Browser Name and version: Chrome Latest
- Operating System and version (desktop or mobile):

**Comment by jcfilben** [comment]
Hi @avantika-codes you can try running `yarn jest -u` to update snapshots, then put up a pull request and we can take a look at the changes

**Comment by avantika-codes** [comment]
Hi @jcfilben, I have created a PR for this issue, when running yarn test || npm test on my local, no test cases fail. Although the pipeline fails in PR.
